scripts/ci/build.sh

```diff
@@ -2,17 +2,28 @@
 set -ex
 
 ROOTDIR="$(dirname "$0")/../.."
+# shellcheck disable=SC1091
```

Comment:
```suggestion
# shellcheck source=scripts/common.sh
```

Reply:
Hum...in this way I got the following error in my machine:
```
In scripts/ci/build.sh line 6:
source "${ROOTDIR}/scripts/common.sh"
       ^----------------------------^ SC1091 (info): Not following: scripts/common.sh was not specified as input (see shellcheck -x).
```

Does it work in yours?

Shellcheck seems to be unable to follow dynamic paths like `${ROOTDIR}/scripts/common.sh`. See [here](https://github.com/koalaman/shellcheck/wiki/SC1091#exceptions)

Reply:
Haven't checked this particular script but it does work for us in other scripts in the same exact situation:
https://github.com/ethereum/solidity/blob/4b0348a9a5e9e70319ef9b2a87fd8ce7ec43ce0f/.circleci/soltest.sh#L44-L45

Maybe it's because of the path? I thought it was relative to the root of the repo but maybe it's not?

> Shellcheck seems to be unable to follow dynamic paths like `${ROOTDIR}/scripts/common.sh`

Yeah, that's what the `source=` is supposed to help with. It tells shellcheck where to find the file. I think that the "Exceptions" section is talking about what can happen when you don't use it.

Reply:
Currently, with `shellcheck disable=SC1091` you're actually getting this error from shellcheck in CI:
```
SC1090: Can't follow non-constant source. Use a directive to specify location.
```
Which basically tells you to use the `source=` directive to tell shellcheck where the file is.

Reply:
Hum...I see. Right, I will change to use the `source` directive;)